Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -174,12 +174,12 @@ WarpXOpenPMDPlot::~WarpXOpenPMDPlot()
 //
 void WarpXOpenPMDPlot::GetFileName(std::string& filename)
 {
-  filename.append(m_OpenPMDFileType).append("/simData");
+  filename.append("/openpmd");
   //
   // OpenPMD supports timestepped names
   //
   if (m_OneFilePerTS)
-      filename = filename.append("_%07T");
+      filename = filename.append("_%05T");
```

Comment:
Like `amrex::Concatenate`, we can now express 99'999 (NEIN-NEIN, NEIN-NEIN-NEIN) iterations max :)

Reply:
Hm, 6-sigits are legit tbh.